app/views/content.js

```diff
@@ -13,21 +13,27 @@ const Content = Mn.LayoutView.extend({
         tracks: '.tracks',
     },
 
-    modelEvents: {
-        'change:currentPlaylist': 'switchPlaylist'
+    onBeforeShow() {
+        this.showChildView('header', new HeaderView({ model: this.model }));
+        this.showChildView('tracks', new TracksView({
+            collection: application.allTracks.get('tracks')
+        }));
+        this.setClass(application.appState.get('currentPlaylist'));
     },
 
-    onBeforeShow () {
-        this.showChildView('header', new HeaderView({ model: this.model }));
-        this.switchPlaylist(null, this.model.get('currentPlaylist'));
+    initialize() {
+         this.listenTo(application.appState,
+            'change:currentPlaylist',
+            (appState, currentPlaylist) => {
+                this.setClass(currentPlaylist);
+            }
+        );
     },
 
-    // only the router change the current playlist
-    switchPlaylist (appState, currentPlaylist) {
-        this.showChildView('tracks', new TracksView({
-            collection: currentPlaylist.get('tracks')
-        }));
-    }
+    setClass(currentPlaylist) {
+        let type = currentPlaylist.get('tracks').type;
+        this.$el.children('.tracks').get(0).className = 'tracks ' + type;
```

Comment:
QuickTip for later use: you can use [`classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) to manipulate classes on an element in a more clever way rather than doing it the pold way with `className` string.

Plus, in this case, you can safely use the jQuery `addClass` as long as `this.$el.children('.tracks')` retruns a jQuery object :).